### DISCUSSION TITLE
An option for saving domains marked for prefetching, and prefetching window in Redis database together with DNS queries

### DISCUSSION THREAD

**Discussion by alpominth** [0xERR0R/blocky#1010] [Ideas]
This idea is related to [#905](https://github.com/0xERR0R/blocky/discussions/905)

This option would give the ability to save the prefetching window and DNS queries marked for prefetching in Redis, because they are stored in memory and don't persist across reboots, saving in Redis would be nice in systems with limited amount of RAM as the data would be written to disk.

And also, the prefetching window and DNS queries marked for prefetching would be shared among other instances of Blocky.

At least for me, it would be a nice feature

**Comment by kwitsch** [comment]
Already part of the redis rework plan in #945 .

**Reply by alpominth** [reply]
Oh, I didn't see that, I'm looking forward for that.

Good luck at the work! ;)
